packages/studio-be/src/core/app/botpress.ts

```diff
@@ -137,8 +137,7 @@ export class Botpress {
       }`
     )
 
-    const maxConcurrentMount = parseInt(process.env.MAX_CONCURRENT_MOUNT || '5')
-    await Promise.map(botsToMount, botId => this.botService.mountBot(botId), { concurrency: maxConcurrentMount })
+    await Promise.map(botsToMount, botId => this.botService.mountBot(botId))
```

Comment:
Why remove the concurrency?

Reply:
Because it starts the migration of 5 bots at the same time and every logs gets all messed up, that's the reason why studio migration logs don't make any sense right now. It makes more sense to mount them one after the other. Eventually the goal is to have the studio to work on one bot at a time